### ISSUE TITLE
Kiosk Admin user interface for constants

### ISSUE THREAD

**Issue by urapadmin** [arch-kiosk/arch-kiosk-office#1912]
Changing the recording configuration only via workstation download and upload is cumbersome, particularly for those who work overseas and have a slow line (me). It should be possible to define a flask-admin user interface for constants.
The only important step is that constants need a new modification timestamp when changed.

**Comment by urapadmin** [comment]
### Kiosk 1.2.8
The UI is there BUT it is only of very limited use and should be used only if you know exactly what you are doing. It has even less guidance than the configuration in the recording system

**Comment by luizaogs** [comment]
So changing a value in Kiosk worked fine, but changing a value in my workstation and synchronizing it does not update that value in the Kiosk constants UI. I am not sure if this could be time zone related.

I changed the help_cm_description field. First I changed it in the Kiosk UI to "Hmmm does this value transfer onto my workstation? I wonder." I got it in the workstation just fine. I then changed it in the workstation to the following: 

<img width="1023" alt="Screenshot 2022-11-07 at 9 35 12 PM" src="https://user-images.githubusercontent.com/67970402/200469353-15d5d41e-13bc-4353-8020-70ab75bfee91.png">

I uploaded and synchronized, so expected that this would now show up in the Kiosk UI. Instead I still get this: 

<img width="1491" alt="Screenshot 2022-11-07 at 9 37 22 PM" src="https://user-images.githubusercontent.com/67970402/200469449-18fd3e54-617d-44c8-b02a-79e93f75b629.png">

And then if I download the workstation again my latest change got erased and I am back to the original change I had made in Kiosk: 

<img width="1027" alt="Screenshot 2022-11-07 at 9 39 20 PM" src="https://user-images.githubusercontent.com/67970402/200469732-1163ca61-612d-4c66-b497-74aafeb53c59.png">


I also haven't looked into value lists yet but I am leaving this alone for now in case it makes it easier for you to look into it. I synchronized around 9:32pm Chicago time (GMT-6), in case that is useful for the logs.

**Comment by urapadmin** [comment]
That is quite possibly a timezone issue, indeed. When you change something on the server, you are using the server's timezone.
When you are changing something in FileMaker you are using your own TimeZone. The result is that your FileMaker change is earlier than that of the server and synchronization will dismiss your change. 
You have to wait until you are in Providence :o) 
Or @lbestock helps here.

**Comment by lbestock** [comment]
Sipping tea until I am awake enough to put on my shoes and go vote, I have tested this in the server's timezone.
I changed the same field @luizaogs did.
It was just a timezone issue - works perfectly! Being able to sort by modification date was very helpful. I can't believe I half configured URAP yesterday before this - it's much more convenient and burns fewer trees.

I also changed a valuelist in Kiosk to see if that ended up in my workstation.
In the configuration itself it looks a little weird, with extra lines between the values in the valuelist:
![image](https://user-images.githubusercontent.com/49291909/200564856-881de6a6-96c9-49ec-9f65-6ee2dab7bbe8.png)

But it is totally as expected functionally creating a record, with the ability to use the value I input via Kiosk.

@urapadmin can we close this or does it need more testing?
